rules/do-you-use-logging-fakes/rule.md

```diff
@@ -0,0 +1,142 @@
+---
+seoDescription: Learn how to use Microsoft's `FakeLogger` in unit tests to verify logging behavior in .NET applications. This guide shows you how to capture log entries, check log levels, and ensure your logging works as expected during testing.
+type: rule
+title: Do you use Logging Fakes?
+uri: do-you-use-logging-fakes
```

Comment:
Rule needs to be linked to one or more categories

Reply:
✅ Done added it to `rules-to-better-testing.md`